test/ModuleInterface/features.swift

```diff
@@ -42,9 +42,12 @@ public func globalAsync() async { }
 @_marker public protocol MP2: MP { }
 
 // CHECK: #if compiler(>=5.3) && $MarkerProtocol
```

Comment:
Shouldn't this be compiler(>=5.4)?

Reply:
It doesn't really matter what version we use here; the point is that the `compiler` clause ensures that what's inside the `#if` region doesn't get parsed.